### PR TITLE
Improve UI for user entries

### PR DESCRIPTION
**Pull request by tliu22**
-Remove "remove chart"
-Create list of user entries with delete buttons
-Quick validation of input field
-Clean up UI elements

### CODE REVIEW

File: static/timeseries.js [hunk 1]
```diff
@@ -33,24 +33,34 @@ function syncExtremes(e) {
   }
 }
 
+function chartContains(user) {
+  var chart1 = $("[data-highcharts-chart='0'").highcharts();var chart1 = $("[data-highcharts-chart='0'").highcharts();
```
Comment (seanlobo) [inline static/timeseries.js]: Is this a typo? It looks like the same line was pasted twice.

File: templates/timeseries.html [hunk 2]
```diff
@@ -10,43 +10,44 @@
...
+    <style>
+      .right {
+  	height: 27px;
+  	width: 20%;
+        vertical-align: middle;
+      }
+      .left {
+  	width: 75%;
+        vertical-align: middle;
+      }
```
Comment (seanlobo) [inline templates/timeseries.html:30]: Looks like the indentation got a little messed up here.

Comment (tliu22) [reply templates/timeseries.html:30]: Indentation seems to be displaying inconsistently on vim vs github.
vimvsgitinconsistency

And when I edit on Github, it also looks different from both vim and the excerpt you posted...

Comment (seanlobo) [reply templates/timeseries.html:30]: Sounds good to me, I'll merge it in.

**Pull request merged**